src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1668,17 +1669,36 @@ private IPreferences CreateOrLoadPreferences(IPreferences preferences)
             return new PreferenceSettings();
         }
 
-        private static void InitializePreferences(IPreferences preferences)
+        private void InitializePreferences()
         {
-            ProtoCore.Mirror.MirrorData.PrecisionFormat = DynamoUnits.Display.PrecisionFormat = preferences.NumberFormat;
-
-            var settings = preferences as PreferenceSettings;
-            if (settings != null)
+            if (PreferenceSettings != null)
             {
-                settings.InitializeNamespacesToExcludeFromLibrary();
+                ProtoCore.Mirror.MirrorData.PrecisionFormat = DynamoUnits.Display.PrecisionFormat = PreferenceSettings.NumberFormat;
+                PreferenceSettings.InitializeNamespacesToExcludeFromLibrary();
+
+                if (string.IsNullOrEmpty(PreferenceSettings.BackupLocation))
+                {
+                    PreferenceSettings.BackupLocation = pathManager.DefaultBackupDirectory;
+                }
+                UpdateBackupLocation(PreferenceSettings.BackupLocation);
             }
         }
 
+        public bool UpdateBackupLocation(string selectedBackupLocation)
+        {            
+            return pathManager.UpdateBackupLocation(selectedBackupLocation);
+        }
+
+        public bool IsDefaultBackupLocation()
+        {
+            return PreferenceSettings.BackupLocation.Replace(@"\\", @"\").Equals(pathManager.DefaultBackupDirectory.Replace(@"\\", @"\"));
```

Comment:
Do we do the replacement because there is a mix of '\\' and `\`?

Reply:
yes, we create it in the PathManager, here:

![image](https://user-images.githubusercontent.com/96534278/228660988-20399ad0-6237-4e58-a4fd-e66123ba072f.png)

Reply:
> @jesusalvino I still do not get why we need to do the replacement call for both strings, is it more like a string to lower case call to make sure we are comparing the same thing?

removed since we can set the default value through the reset button